adlanluiskoslopp/Formulario.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE HTML>
+<html>
+	<head>
+		<meta charset="UTF-8">
+		<title>Crie Seu Perfil</title>
+		
+		<style type="text/css">
+			a:link{
+				text-decoration: none;
+				color: #0099ff;	
+			}
+			
+			a:visited {
+				text-decoration: underline;
+				color: #ff0066;
+			}
+
+			a:hover{
+				text-decoration: underline, overline;
+				color: #000000;
+				background: #ffff00;
+			}
+		</style>
+	
+	</head>
+	
+	<body bgcolor="#3333ff">
+		<main>
+			<nav>
+				<table border="1">
+					<thead>
+						<tr>
+							<th>Voltar</th>
+							<th>Ler Termos</th>
+						</tr>
+					</thead>			  
+					<tbody>
+						<tr>
+							<td><a href="PaginaInicial.html">PaginaInicial</a></td>
+							<td><a href="TermosDeUso.html" target="blank">Termos de Uso</a></td>
+						</tr>
+					</tbody>
+				</table>
+			
+			</nav>
+			<h1>Crie aqui seu Perfil para Página</h1>		
+		</main>
+		
+		<form>
+			<label id="nome">Digite seu Nome</label>
+			<input for="nome" type="texto" placeholder="Escerva seu nome"/>
```

Comment:
o valor do atributo `type` deve ser em inglês, então não é `texto` e sim `text`. O atributo `for` deve estar no label e o `id` no `input` (o label é `para/for` um campo com `id` x). Deve corrigir isso em todas as ocorrências e em todas as páginas. E o valor do `type` deve ser todo em minusculo, em alguns campos você colocou como maiusculo.